spec/unit/type/file_spec.rb

```diff
@@ -485,17 +488,19 @@
     describe "and a source is set" do
       it "should pass the already-discovered resources to recurse_remote" do
         file[:source] = File.expand_path(__FILE__)
-        allow(file).to receive(:recurse_local).and_return(:foo => "bar")
-        expect(file).to receive(:recurse_remote).with(:foo => "bar").and_return([])
+        allow(child).to receive(:[]).with(:path).and_return(name)
```

Comment:
Are these related to a RuboCop issue?

Reply:
I updated the commit summary to describe the test change.